components/wifi/esp8266-driver/ESP8266/ESP8266.cpp

```diff
@@ -482,12 +483,12 @@ int8_t ESP8266::rssi()
     return rssi;
 }
 
-int ESP8266::scan(WiFiAccessPoint *res, unsigned limit, scan_mode mode, unsigned t_max, unsigned t_min)
+int ESP8266::scan(WiFiAccessPoint *res, unsigned limit, scan_mode mode, std::chrono::duration<unsigned, std::milli> t_max, std::chrono::duration<unsigned, std::milli> t_min)
```

Comment:
There's `using std::chrono` in the beginning of the file, so perhaps you can shorten the type declarations (or drop the `using`)?

Reply:
Yep. Declarations in the header have to be verbose though, which is annoying.

Doubly so, as you can't use `50ms` in header files as you have to avoid namespaces. Have to say `std::chrono::milliseconds(50)`. :(